src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -469,4 +472,91 @@ void VehicleAirData::PrintStatus()
 	}
 }
 
+bool VehicleAirData::BaroGNSSAltitudeOffset()
+{
+	static constexpr float kEpvReq = 8.f;
+	static constexpr float kDeltaOffsetTolerance = 4.f;
+	static constexpr uint64_t kLpfWindow = 2_s;
```

Comment:
```suggestion
	static constexpr float kLpfTimeConstant = 2.f;
```
might be simpler to directly directly as a float in seconds than having to multiply and cast later.

Reply:
For the "time past since" conditions further below, I would need to cast it the other way around. Since this part gets called more frequently, I'll keep it like this.